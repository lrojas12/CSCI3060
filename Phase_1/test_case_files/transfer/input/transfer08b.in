login
admin
transfer
Tarzan
00001
10000
logout
quit