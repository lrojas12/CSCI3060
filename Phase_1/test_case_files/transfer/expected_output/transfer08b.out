Welcome to Watermelon Banking System
Please login to begin or type in “help” for more information.
Mode in which you wish to login as:
Account holder’s name:
You are currently logged in as Bill
Bank Account Number: 00014
Balance: $1000.00
Transaction Payment Plan: non-student
Status: active
Enter your account number:
Enter destination account number:
ERROR: Newly created accounts may not send receive.
Please try again in 24 hours.
You have successfully logged out of your account.